SYCL/ExtraTests/README.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+"ExtraTests" directory contains infrastructure for picking up LIT style tests
+from another external directory passed in SYCL_EXTRA_TESTS_SRC.
```

Comment:
Could you clarify what is the difference with [External](https://github.com/intel/llvm-test-suite/tree/intel/SYCL/External) tests, please?
It looks to me that both are intended to run LIT-based tests located out-of-tree. Right?

Reply:
@pvchupin you are right about ExtraTests and External test.
 - ExtraTests can be used to run intel/llvm in-tree LIT tests from llvm-test-suite framework.
 - External tests allow to download/build/run external applications (e.g. RSBench from https://github.com/ANL-CESAR/RSBench.git). The tests were marked as XFail because benchmark internal data verification failed (it is successfully build and code is executed without crashes).

Reply:
Marked both as experimental. We should probably remove at some later point if no use.